Dockerfile

```diff
@@ -1,16 +1,24 @@
-FROM golang:1.9.6 AS build
+FROM golang:1.10-alpine AS build
```

Comment:
For the build to be reproducible, please include the patch number: `1.10.3-alpine3.8`.